app/UI/setting.tsx

```diff
@@ -151,6 +160,14 @@ export class RelaySetting extends Component<RelaySettingProp, RelaySettingState>
         return _relayStatus;
     }
 
+    computeRecommendedRelaysStatus() {
+        // remove the relay that is already in the relay list
+        const _recommendedRelaysStatus = this.state.recommendedRelaysList.filter((r) => {
+            return this.state.relayStatus.find((rs) => rs.url == r.url) == undefined;
+        });
+        return _recommendedRelaysStatus;
+    }
+
```

Comment:
Status only exists for selected relays. No need for recommendation/discovery list